packages/sdk/package.json

```diff
@@ -30,7 +30,7 @@
         "@river-build/dlog": "workspace:^",
         "@river-build/encryption": "workspace:^",
         "@river-build/generated": "workspace:^",
-        "@river-build/mls-rs-wasm": "^0.0.11",
+        "@river-build/mls-rs-wasm": "^0.0.13",
```

Comment:
Please make sure that you also bump `mls-rs-wasm-node` in `packages/stress/package.json`.  Not sure if this has to be as part of this PR or a different PR, but the version of those two packages should be kept in sync.